package.json

```diff
@@ -19,12 +19,14 @@
     "url": "https://github.com/DetachHead/eslint-config.git"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.13.0",
-    "@typescript-eslint/parser": "^4.13.0",
-    "eslint": "^7.17.0",
-    "eslint-config-prettier": "^7.1.0",
+    "@typescript-eslint/parser": "^5.9.0",
+    "eslint": "^8.6.0",
```

Comment:
Hey, I saw your PR in https://github.com/eslint/eslint/issues/3458. I wonder is it really necessary to put `eslint` into dependencies? `eslint` is clearly a huge package (https://bundlephobia.com/package/eslint@8.6.0) and all consumers of the shareable config will have `eslint` 🤔

Reply:
This package should only ever be used as a `devDependency`, so I don’t see the issue because you can’t use it without eslint